### PR TITLE
Check ide-vscode for blockers in release script

### PR DESCRIPTION
**Pull request by alex-chew**

### CODE REVIEW

File: Scripts/prepare_release.py [hunk 1]
```diff
@@ -290,10 +290,16 @@ def _head_up_to_date(cls) -> bool:
     def _no_release_blocking_issues() -> bool:
         progress("Checking... ", end="")
         HEADERS = {"Accept": "application/vnd.github.v3+json"}
-        ENDPOINT = 'https://api.github.com/repos/dafny-lang/dafny/issues?labels=severity%3A+release-blocker'
-        with urlopen(Request(ENDPOINT, data=None, headers=HEADERS)) as fs:
-            response = fs.read().decode("utf-8")
-        return json.loads(response) == []
+        ENDPOINTS = (
+            'https://api.github.com/repos/dafny-lang/dafny/issues?labels=severity%3A+release-blocker',
+            "https://api.github.com/repos/dafny-lang/ide-vscode/issues?labels=severity%3A+release-blocker",
```
Comment (robin-aws) [inline Scripts/prepare_release.py:295]:
```suggestion
            'https://api.github.com/repos/dafny-lang/ide-vscode/issues?labels=severity%3A+release-blocker',
```

Comment (alex-chew) [reply Scripts/prepare_release.py:295]: I actually made the opposite change (single quote to double quote, on the other endpoint) for consistency with the rest of the file.

**Review by robin-aws — COMMENTED** [review]
Good call. Were you able to test it with a `--dry-run`?

**Comment by alex-chew** [comment]
I've tested the `_no_release_blocking_issues` on its own in a shell, but a `--dry-run` won't work as written because the other checks won't pass (there's nothing in `docs/dev/news`, it's not on master, etc.).

**Review by robin-aws — APPROVED** [review]
🚀

**Pull request merged**